### DISCUSSION TITLE
Gatsby Build Service w/o restoring cache?

### DISCUSSION THREAD

**Discussion by ghost** [gatsbyjs/gatsby#36626] [Help]
How to use Gatsby Build Service and don't use cache option? Build fails when pushed to Github. But work during manual build without cache.
Netlify build works fine. Gatsby Cloud don't. 
Version of gatsby: "^2.24.63",

![image](https://user-images.githubusercontent.com/20955623/190686244-7f8e2499-5504-45bd-a7bc-5293c3dfe0d6.png)
![image](https://user-images.githubusercontent.com/20955623/190686263-e3277de6-47b4-4da5-ae64-baee944eaa9c.png)
![image](https://user-images.githubusercontent.com/20955623/190686298-b790e91d-3cd6-4596-b97d-d74c62a64db9.png)

Fails during automatic build.

But succeeds during manual build. with option "clear cache and build main". 
![image](https://user-images.githubusercontent.com/20955623/190686410-75babf7b-76f2-4f4b-9ca3-1e3b12f3feed.png)
https://github.com/stephyswe/gatsby-learn-reactjs

Shows the issue at hand. One broken (cache), one working ( no cache) 
![Screenshot 2022-09-16 184240](https://user-images.githubusercontent.com/20955623/190688453-6f8d6262-5866-4975-9a05-f8b33ba8c555.png)

**Comment by ghost** [comment]
Tells everything i guess :) . Same code, one broken (cache), one working ( no cache) 

![Screenshot 2022-09-16 184240](https://user-images.githubusercontent.com/20955623/190688453-6f8d6262-5866-4975-9a05-f8b33ba8c555.png)

**Comment by LekoArts** [comment]
If your site is "broken" with restoring the cache, then something in your site is incorrect. Disabling the cache isn't really an option. Also, you should use Gatsby v4 and not Gatsby v2 as the latter is a couple of years old.
